docs/stackit_argus_scrape-configs.md

```diff
@@ -29,5 +29,6 @@ stackit argus scrape-configs [flags]
 ### SEE ALSO
 
 * [stackit argus](./stackit_argus.md)	 - Provides functionality for Argus
+* [stackit argus scrape-configs create](./stackit_argus_scrape-configs_create.md)	 - Creates a Scrape Config Job for an Argus instance
```

Comment:
Along the PR, some points job is written as **J**ob and some places it is written as **j**ob.  
I think it will be better if we have it start with lower case everywhere.